build.gradle

```diff
@@ -1,22 +1,35 @@
 // Required by spotless. https://github.com/diffplug/spotless/issues/747
 buildscript {
+    ext {
+        // plugin versions
+        vSpringBoot = '3.3.4'
```

Comment:
spring boot bumped from 3.3.3 to 3.3.4
dep management plugin bumped from 1.1.5 to 1.1.6